lab_1/main.py

```diff
@@ -25,7 +32,10 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
+    for i in tokens:
+        if i in stop_words:
+            tokens.pop(tokens.index(i))
```

Comment:
try this:
tokens = ['the', 'is', 'a', 'the']
stop_words = ['the', 'is']

The expected result: ['a']